lib/model/QubitDigitalObject.php

```diff
@@ -2010,6 +2010,17 @@ public function createRepresentations($usageId, $connection = null)
           // Extract text
           $this->extractText($connection);
         }
+        else if ($this->canThumbnail())
```

Comment:
I think it would be better to wrap the whole TEXT case body in `if ($this->canThumbnail())`, similar to the IMAGE case.  If the PDF can't be thumbnailed then none of the derivative creation methods will work.